sprockets/http/__init__.py

```diff
@@ -77,8 +90,16 @@ def run(create_application, settings=None, log_config=_unspecified):
 
     port_number = int(app_settings.pop('port', os.environ.get('PORT', 8000)))
     num_procs = int(app_settings.pop('number_of_procs', '0'))
-    server = runner.Runner(create_application(**app_settings))
+    app = create_application(**app_settings)
+
+    if sentry_sdk is not None:
+        sentry_sdk.init(
```

Comment:
Should allow the application to override these kwargs, e.g. adding a `before_send`

Reply:
Since we're already opinionated about `app.settings`, maybe override with the contents of `app.settings['sentry_sdk']` if it exists